src/app/models/Question.ts

```diff
@@ -36,6 +37,18 @@ export class Question extends AbstractModel implements IQuestion {
   @Updater<Question>()
   public updater?: User;
 
+  public static generate(id?: Id): IQuestion {
+    return {
+      id: modelData.id(id),
+      text: modelData.notes(), // TODO This may not be the correct type
+      data: modelData.notes(), // TODO This may not be the correct type
```

Comment:
data will be an arbitrary object (correct)

text will be a html/textual blurb (incorrect)